src/components/governance/PageProposals.vue

```diff
@@ -35,7 +35,9 @@
           @click.native="onPropose"
         />
       </div>
+      <TmDataLoading v-if="$apollo.loading" />
       <TableProposals
+        v-else
         :proposals="proposals"
         :loading="$apollo.queries.proposals.loading"
```

Comment:
is this used?

Reply:
@colw

Reply:
There is a `v-if` for loading above it?

Reply:
Not sure I understand.

Reply:
`:loading="$apollo.queries.proposals.loading"` is not used `TableProposals` doesn't have the property

Reply:
I see the loading on the TableProposals though is superfluous. I will remove that.